src/Adapters/__mockData__/3DScenesConfiguration.jsonc

```diff
@@ -66,18 +70,21 @@
                         "url": "https://cardboardresources.blob.core.windows.net/cardboard-mock-files/TruckBoxesEnginesPastmachineReduced.gltf",
                         "extensionProperties": {}
                     }
-                ]
+                ],
+                /* the polling configuration will fetch data as quickly as allowed given the number of twins on the screen */
+                "pollingConfiguration": {
+                    "pollingStrategy": "Realtime",
```

Comment:
Can we define an enum type in the config schema so that the polling strategy is no longer a type string?